### ISSUE TITLE
[NEW ALGORITHM]: Find the Only Non-repeating Element in an Array

### ISSUE THREAD

**Issue by Jivanjamadar** [AlgoGenesis/C#1637]
## Issue will be closed if:

> 1) You mention more than one algorithm. You can create a separate issue for each algorithm once the current one is completed.  
> 2) You propose an algorithm that is already present or has been mentioned in a previous issue.  
> 3) You create a new issue without completing your previous issue.

**Note:** These actions will be taken seriously. Failure to follow the guidelines may result in the immediate closure of your issue.

---

### Name:  Find the Only Non-repeating Element in an Array
```[NEW ALGORITHM]```  

### About:  
Given an array of integers where every element appears twice except for one element that appears only once, your task is to find that unique element.

Example:
Input: arr = [4, 1, 2, 1, 2]
Output: 4
Propose a new algorithm to be added to the repository

---

### Labels:  
```new algorithm, gssoc-ext, hacktoberfest, level1```

---

### Assignees:  
- [X] Contributor in GSSoC-ext
- [X] Want to work on it

**Comment by Jivanjamadar** [comment]
@pankaj-bind assign this issue to me
